apps/integration_tests/integration_test_fhir_resources.py

```diff
@@ -208,6 +208,56 @@ def _call_health_external_endpoint(self, v2=False):
             pass
         self.assertEqual(msg, "all's well")
 
+    @override_switch('require-scopes', active=True)
+    def test_health_bfd_endpoint(self):
+        self._call_health_bfd_endpoint(False)
+
+    @override_flag('bfd_v2_flag', active=True)
+    @override_switch('require-scopes', active=True)
+    def test_health_bfd_endpoint_v2(self):
+        self._call_health_bfd_endpoint(True)
+
+    def _call_health_bfd_endpoint(self, v2=False):
+        client = APIClient()
+        # no authenticate needed
+        response = client.get(self.live_server_url + "/health/bfd_v2" if v2 else "/health/bfd")
```

Comment:
I think we should match our current style with v1, v2 instead of creating a new pattern here.
For example:
`/v1/health/bfd`
`/v2/health/bfd`

Reply:
Another option could be just including both in a single health check (bfd v1 & v2).
It does bring up the question though - why do we need specific version checking in a simple health check?

Reply:
It's worth asking BFD if they think their V1 and V2 could ever have separate availability concerns, because if thats possible, I'd rather not have both health checks lumped into one.

Reply:
Good idea, folks. Other than this concern overall looks good!

Reply:
@nbragdon @njdister Will see if can put api version component at the base of the url, also will ask BFD if ever v1 meta vs v2 meta would demonstrate different behavior.

when designing v2, the general guideline is to totally separate v1 and v2 end points, since over time, v1 will be deprecated, and v2 evolve etc., for bfd health, v1 bfd check mapped to bfd v1 metadata, and v2 bfd check mapped to bfd v2 metadata, and it could occur that v1 metadata failed while v2 metadata return OK and vice versa.

Reply:
I posted the question to BFD users, we'll see what they say.

Reply:
BFD confirmed V1 and V2 are separate servelets that should be tested separately.